src/utils/hex.cairo

```diff
@@ -18,6 +18,19 @@ pub fn from_hex(hex_string: ByteArray) -> ByteArray {
     bytes
 }
 
+pub fn from_hex_rev(hex_string: ByteArray) -> ByteArray {
```

Comment:
You'd need to return `Hash` though (so that you can render `txid: from_hex_rev("00000000"),` in the script).
Maybe `from_hash_rev` would be a more suitable name 🤔 

So just for the context, this helper is supposed to be a more explicit alternative to `0x{hex}_u256.into()` (which already led to several issues). It can be used for transaction hashes, block hashes, and merkle roots in the tests.

Reply:
Shouldn't `ByteArray` -> `Hash` conversion be handled by `TryInto` implemented in `hash.cairo`?

Reply:
It's more about convenience, so that you don't have to write `from_hash_rev().try_into().unwrap()`

Reply:
Fair enough. So function named `from_hash_rev(...) -> Hash` should be added (I am writing it here to make it clear what is missing in the pr)?

Reply:
Given that we want to move away from code generated scripts, we don't need to modify the get_block.py, just use this new helper instead of 0x12345_u256.into() in unit tests

Reply:
Its a detail but `to_hash_rev` or `hex_to_hash_rev` seems a little more logical name

Reply:
Agree, `hex_to_hash_rev` sounds comprehensive